### PR TITLE
Support for testcontainers.ContainerCustomizer

### PR DESCRIPTION
**Pull request by uchr**
Update the module according to the testcontainer [guideline](https://golang.testcontainers.org/modules/).

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,76 +1,82 @@
 module github.com/nebius/testcontainers-ytsaurus
 
-go 1.21
+go 1.22
```
Comment (l0kix2) [inline go.mod:3]: Please update version in the gh workflows https://github.com/nebius/testcontainers-ytsaurus/blob/main/.github/workflows/test.yml#L22
Currntly check is failing https://github.com/nebius/testcontainers-ytsaurus/actions/runs/9253263724/job/25453525829?pr=3

**Review by l0kix2 — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1 +1,53 @@
...
+## Introduction
+
+The Testcontainers module for [YTsaurus](https://ytsaurus.tech/).
+
+## Adding this module to your project dependencies
+
+Please run the following command to add the YTsaurus module to your Go dependencies:
+
+```
+go get github.com/testcontainers/testcontainers-go/modules/ytsaurus
```
Comment (l0kix2) [inline README.md:12]: I don't think we should try to be added to the `testcontainers/testcontainers-go`repo, so url it should be `nebius/testcontainers-ytsaurus`.

**Review by l0kix2 — CHANGES_REQUESTED** [review]
Let's fix the `go get` line. Everything else looks good.

File: README.md (hunk 2 above)
Comment (uchr) [reply README.md]: Yeah. Missed that one.

**Pull request merged**